pkg/cluster/nodes/node.go

```diff
@@ -234,6 +234,14 @@ func (n *Node) WriteFile(dest, content string) error {
 	return n.Command("cp", "/dev/stdin", dest).SetStdin(strings.NewReader(content)).Run()
 }
 
+// ImageInspect return low-level information on containers images inside a node
+func (n *Node) ImageInspect(containerNameOrID string) ([]string, error) {
+	cmd := n.Command(
+		"crictl", "-r", "/var/run/containerd/containerd.sock", "inspecti", containerNameOrID,
```

Comment:
we don't need to set the socket, kind images have included this pre-configured for a while. not specifying it buys us free future {insert-cri-here} support for this feature

```
$ docker exec kind-control-plane crictl ps
CONTAINER ID        IMAGE               CREATED             STATE               NAME                      ATTEMPT             POD ID
33f1589cc2129       b3b94275d97cb       22 hours ago        Running             coredns                   0                   7ef4face252c1
c9a518029909c       b3b94275d97cb       22 hours ago        Running             coredns                   0                   2863336690554
a7a5abebcf708       f227066bdc5f9       22 hours ago        Running             kindnet-cni               0                   7b5c8553efd43
04ecc7a8c3c30       19bb968f77bba       22 hours ago        Running             ip-masq-agent             0                   f28561ce51146
6efcff288f6b7       6d859c5ba0871       22 hours ago        Running             kube-proxy                0                   50869b3535504
176dbdf4dd874       b8df3b177be23       22 hours ago        Running             etcd                      0                   dde1d7249abb4
69b32e5f5e6dd       b2e25e147ed1c       22 hours ago        Running             kube-apiserver            0                   7c095b1397162
30fc0b2cf24e6       b91e0e16786e3       22 hours ago        Running             kube-scheduler            0                   326d485d5cb91
081b277edb8ef       3eb89ae87afc9       22 hours ago        Running             kube-controller-manager   0                   12df50441fdbf
```

Reply:
done